amun/base/deploymentconfig.yaml

```diff
@@ -33,6 +33,16 @@ spec:
               value: "0"
             - name: THOTH_SENTRY_IGNORE_LOGGER
               value: "connexion.decorators.validation"
+            - name: THOTH_AMUN_API_URL
+              valueFrom:
+                configMapKeyRef:
+                  name: amun
+                  key: amun-api-url
+            - name: THOTH_AMUN_DEPLOYMENT_NAME
+              valueFrom:
+                configMapKeyRef:
+                  name: amun
+                  key: deployment-name
```

Comment:
is this a replacement for this https://github.com/thoth-station/thoth-application/blob/e478b846cf6242b105aacc634e8147ca077c253e/amun/base/deploymentconfig.yaml#L91
or both are to be provided ?

Reply:
Good catch! :+1: I missed that one. Adjusted.

Reply:
can you point me to the code where all these would be used by, i m kinda forgetting how this would be used in whole process.
Thanks.

Reply:
ah i see it here: https://github.com/thoth-station/amun-api/pull/567/files
thanks

Reply:
It's propagated to the specification document stored on Ceph: https://github.com/thoth-station/amun-api/pull/567/files This specification document specifies the whole request to Amun and states all the input with some metadata. As we want to notify that an inspection finished via a Kafka message, we need to propagate this information in the message somewhere:

https://github.com/thoth-station/messaging/pull/270

It would be also good to have this stated in the specification document if we plan to have multiple Amun deployments (multiple clusters) to aggregate results.